src/sbuf/SBuf.cc

```diff
@@ -258,7 +258,7 @@ SBuf::vappendf(const char *fmt, va_list vargs)
     va_copy(ap, vargs);
     sz = vsnprintf(space, spaceSize(), fmt, ap);
     va_end(ap);
-    Must2(sz >= 0, "vsnprintf() output error");
+    Must3(sz >= 0, "vsnprintf() succeeds", Here());
```

Comment:
A non-negative result is how vsnprintf() manual pages define success. We can say "wrote or intended to write zero or more bytes" instead, but I think the current version is better.